register-computer/main.py

```diff
@@ -98,6 +102,47 @@ def __connect_to_activedirectory():
 def __generate_password(length=40):
     return str(uuid.uuid4()) + "-" + str(uuid.uuid4())
 
+def __shorten_computer_name(computer_name, gce_instance):    
+    # Look for MIG related metadata
+    has_metadata = ("metadata" in gce_instance.keys() and
```

Comment:
Move into `if` condition instead of creating a temp variable

Reply:
done